README.md

```diff
@@ -47,12 +47,13 @@ end
   config.registration.unregister_on_halt = false
 ```
 
-### subscription-manager Configuration
+- **manager** select the registration manager implementation. By default the plugin will use the implementation for the subscription-manager's register command, you can however change that by setting the option to a different implementation:
```

Comment:
What do you think about replacing 'implementation' with 'provider'?

**manager** select the registration manager provider. By default the plugin will use the `subscription_manager` manager provider, you can however change that by setting the option to a different provider: